config/application.rb

```diff
@@ -69,6 +69,8 @@ class Application < Rails::Application
     config.good_job.enable_cron = true
     config.good_job.max_threads = IdentityConfig.store.good_job_max_threads
     config.good_job.queues = IdentityConfig.store.good_job_queues
+    config.good_job.preserve_job_records = false
+    config.good_job.queue_select_limit = 5_000
```

Comment:
I'd make this configurable now and save future us a PR

Reply:
Yep, thanks for the nudge and saving future versions of me some headache, added in [8b8f2d8](https://github.com/18F/identity-idp/pull/7196/commits/8b8f2d8fbe609e893ee93cc668293a62ec323b38)